elements/pfe-navigation/documentation/implementation.md

```diff
@@ -0,0 +1,227 @@
+# Implementation
+
+> If there are any questions, issues, feel there's something missing, please contact the CPFED Group.
+
+## Adding scripts and styles
+Follow the [instructions from Patternfly Elements to get the javascript added to the page](https://patternfly.github.io/patternfly-elements/getting-started/).
+
+Add the fallback stylesheet, which covers non-JS users and initial page load nicities:
+```html
+<link rel="stylesheet" href="PATH/TO/pfe-navigation--lightdom.css" type="text/css">
+```
+
+## Adding HTML
+
+The bare minimum skeleton HTML is:
+
+```html
+<!-- These links should be directly after <body> -->
+<a href="#pfe-navigation" class="visually-hidden">Skip to navigation</a>
+<!-- !! Update anchor link to main/content -->
+<a href="#ADD-ID-TO-MAIN" class="visually-hidden">Skip to content</a>
+
+<pfe-navigation id="pfe-navigation">
```

Comment:
Rather than duplicate the pfe-navigation tag and the nav tag which are essentially communicating the same semantics, why not use `<pfe-navigation  role="navigation" aria-label="Main navigation">`?  This gives you more stylistic control over the children and makes it easier to use slots for the sections below instead of relying on developers to use the right classes in the right order.